### ISSUE TITLE
ERROR kuhl_m_sekurlsa_acquireLSA ; Logon list

### ISSUE THREAD

**Issue by CE1CECL** [gentilkiwi/mimikatz#350]
Running on Windows 10 Enterprise Build 21301
I ran the newest mimikatz as of this issue!

**Comment by gentilkiwi** [comment]
This is not (yet) a Windows release version

**Comment by st4rry** [comment]
I've experienced with the Error too. But now it's okay with the last fix. 
The version 2.2.0 20210810-2 2.2.0 20210810-2 Windows 365 Web passwords junk-fix:  is working on Windows11. 
https://github.com/gentilkiwi/mimikatz/releases/

**Comment by CE1CECL** [comment]
Are you sure it's this error @st4rry ?
I'm still getting it. And I have to use pypykatz b/c thats been working
Still using Windows 10 Enterprise Build 21301 x64 btw

**Comment by st4rry** [comment]
Hi @CE1CECL , Yeah I'm sure. I have been experiencing this Login List Error for a long time and tried both downgraded versions and latest releases. Read through all forums and found you mentioned about the same error here. And I found one of the forums where @gentilkiwi mentioned about Secure Kernel and injection and linked his repo here and I tried the latest fix for 365 Web passwords and figured out it also fix the error I'm having. 

But, I'm just running on a fresh installed windows 11, and it's on VMware, so there's no other secure boots, IOMM , credential guards stuffs on the windows 11 that I'm running the latest release. 

And I've made a video here for you : https://youtu.be/ksMi3p-vvEk
